packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -402,6 +405,16 @@ class Dropdown extends AutoControlledComponent<Extendable<DropdownProps>, Dropdo
             )
           }}
         </Downshift>
+        <Portal open={!!this.props.getA11ySelectionMessage}>
+          <div
+            role="status"
+            aria-live="polite"
+            aria-relevant="additions text"
+            style={screenReaderContainerStyles}
+          >
+            {this.state.a11ySelectionStatus}
```

Comment:
nit: `this.state` is destructured at line 312, you can move `a11ySelectionStatus ` there